recipes/cpprestsdk/all/conanfile.py

```diff
@@ -55,8 +55,8 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("boost/1.77.0")
-        self.requires("openssl/1.1.1k")
+        self.requires("boost/1.78.0")
```

Comment:
why bump to 1.78.0? I think we previously (see #8944) agreed it's not needed for two reasons:
- people experiences segfault with boost versions different from 1.77
- downstream consumers already may override to 1.78 (or any other version)

Reply:
Good question :thinking:

Reply:
Excellent question!

I was under the impression #8856 fixed the underlying issue. The override didnt work and mixing cause problems as I understand it.

I suspect most Conan users ride latest so it seems odd to hold back.
Other are probably on recipe revision and now they can choice the boost version they want (since 1.77 is published now).